lib/sisimai/bite/json/sendgrid.rb

```diff
@@ -0,0 +1,79 @@
+module Sisimai::Bite::JSON
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->